blocks/small-promo-block/features/small-promo/default.jsx

```diff
@@ -35,8 +35,9 @@ const SmallPromo = ({ customFields }) => {
 	} = customFields;
 	const { registerSuccessEvent } = useComponentContext();
 	const { arcSite, isAdmin } = useFusionContext();
-	const { searchableField } = useEditableContent();
+	const { editableContent, searchableField } = useEditableContent();
 	const shouldLazyLoad = lazyLoad && !isAdmin;
+	const editable = true;
```

Comment:
editable will always be true, so it's not needed...

Reply:
@nschubach  Removed editable